forge-gui/res/cardsfolder/upcoming/ajani_sleeper_agent.txt

```diff
@@ -0,0 +1,17 @@
+Name:Ajani, Sleeper Agent
+ManaCost:1 G PGW W
+Types:Legendary Planeswalker Ajani
+Loyalty:4
+K:Compleated
+A:AB$ Dig | Cost$ AddCounter<1/LOYALTY> | Planeswalker$ True | DigNum$ 1 | Reveal$ True | ChangeNum$ All | ChangeValid$ Creature,Planeswalker | DestinationZone$ Hand | DestinationZone2$ Library | LibraryPosition2$ -1 | RememberChanged$ True | SubAbility$ DBDig | SpellDescription$ Reveal the top card of your library. If it's a creature or planeswalker card, put it into your hand. Otherwise, you may put it on the bottom of your library.
+SVar:DBDig:DB$ Dig | DigNum$ 1 | DestinationZone$ Library | Optional$ True | LibraryPosition$ -1 | LibraryPosition2$ 0 | ConditionCheckSVar$ X | ConditionSVarCompare$ EQ0 | SubAbility$ DBCleanup
```

Comment:
Use `ConditionDefined$ Remembered | ConditionNotPresent$ Card` to avoid X SVar

Reply:
Replaced the condition check to match suggested. Thank you!